spec.go

```diff
@@ -129,7 +129,9 @@ type Artifacts struct {
 	Manpages map[string]ArtifactConfig `yaml:"manpages,omitempty" json:"manpages,omitempty"`
 	// Directories is a list of various directories that should be created by the RPM.
 	Directories *CreateArtifactDirectories `yaml:"createDirectories,omitempty" json:"createDirectories,omitempty"`
-	// TODO: other types of artifacts (systtemd units, libexec, configs, etc)
+	// ConfigFiles is a list of files that should be marked as config files in the RPM.
```

Comment:
Missed the `in the RPM` part in the last PR.
Should be `in the package`

Reply:
*doh* updated this and the directories string.